### ISSUE TITLE
clear() method

### ISSUE THREAD

**Issue by iddan** [Level/levelup#648]
A DB method for removing all the keys and values. Currently doing it like:
```javascript
db.createKeyStream().on("data", key => {
    db.del(key);
});
```

**Comment by vweevers** [comment]
It's planned (https://github.com/Level/abstract-leveldown/issues/236, `levelup` would wrap this), we didn't get around to it yet.

**Comment by vweevers** [comment]
Released in 4.2.0 - please see the caveats in the [readme](https://github.com/Level/levelup#clear).
